src/components/TodoModal/TodoModal.tsx

```diff
@@ -0,0 +1,143 @@
+import { useState, FC, useEffect } from "react";
+import { DateTime } from "luxon";
+import { FaPlus } from "react-icons/fa";
+
+import "./TodoModal.scss";
+import TodoInput from "../Inputs/TodoInput";
+import {
+  nextDay,
+  formatLocaleToISO,
+  formatISOToLocale,
+} from "../../utils/dateUtils";
+
+type Props = {
+  start: string;
+  end: string;
+  onSave: (arg1: string, arg2: string) => void;
+  inputTitle: string;
+  setInputTitle: (value: string) => void;
+};
+
+const TodoModal: FC<Props> = ({
+  start,
+  end,
+  onSave,
+  inputTitle,
+  setInputTitle,
+}) => {
+  const [startDate, setStartDate] = useState<string>("");
+  const [endDate, setEndDate] = useState<string>("");
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  useEffect(() => {
+    setStartDate(start);
+    setEndDate(end);
+  }, [isOpen]);
+
+  function handleStartDateChange(e: React.ChangeEvent<HTMLInputElement>) {
+    const startDateObj: DateTime = DateTime.fromISO(e.target.value);
+    const endDateObj: DateTime = DateTime.fromFormat(
+      endDate,
+      "dd.MM.yyyy HH:mm"
+    );
+
+    setStartDate(startDateObj.toFormat("dd.MM.yyyy HH:mm"));
+
+    if (startDateObj > endDateObj) {
+      const newEndDate: string = startDateObj
+        .plus({ days: 1 })
+        .toFormat("dd.MM.yyyy HH:mm");
+      setEndDate(newEndDate);
+    }
+  }
+
+  function handleEndDateChange(e: React.ChangeEvent<HTMLInputElement>) {
+    setEndDate(formatISOToLocale(e.target.value));
+  }
+
+  function handleSave() {
+    onSave(startDate, endDate);
+    setIsOpen(false);
+  }
+
+  function handleClose() {
+    setIsOpen(false);
+  }
+
+  return (
+    <div>
+      <button
+        className="modal__btn-add"
+        onClick={() => setIsOpen(true)}
```

Comment:
Why do you have function handleClose but don't have handleOpen instead of logic in JSX ?